test/parallel/test-crypto-hash-hmac-regression.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+if (!common.enoughTestMem)
+  common.skip('skip on low memory machines');
+
+const {
+  Buffer,
+  kMaxLength
+} = require('buffer');
+
+const {
+  createHash,
+  createHmac
+} = require('crypto');
+
+const kData = Buffer.alloc(kMaxLength);
```

Comment:
This fails on a few CI hosts because of the large size, maybe you can just wrap this in a `try/catch` and skip the test if that throws a `RangeError`?

Reply:
Yep, that's exactly what I'm doing... although I'm just skipping if any error is thrown here.